force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -252,11 +286,11 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String createAndGetBookDetails() {
         // Create a Book object with title "A Brief History of Time" and author "Stephen Hawking"
-
+            Book historyBook1 = new Book('A Brief History of Time', 'Stephen Hawking');
         // Call the getBookDetails method on the book object to get the details of the book
-
+        String historyBookDetails = historyBook1.getBookDetails();
         // Return the details of the book
-        return null; // Replace null with the variable you used to store the result
+        return historyBookDetails; // Replace null with the variable you used to store the result
```

Comment:
exellent. As a way to think about how your code is structured. Your method returns a string amd the method getBookDetails returns a string. 

So you could simple do this. 

   return new Book('A Brief History of Time', 'Stephen Hawking').getBookDetails();